src/test/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilderPipelineIT.java

```diff
@@ -169,8 +174,68 @@ public void testMalformedDeclarativePipelineFails() throws Exception {
     dumpLog(LOGGER, run);
   }
 
+  @Test
+  public void testNoNamespaceDeclarativePipelineDeploysProperly() throws Exception {
+    envVars.put("MANIFEST_PATTERN", TEST_DEPLOYMENT_MANIFEST);
+    WorkflowJob testProject =
+        jenkinsRule.createProject(WorkflowJob.class, formatRandomName("test"));
+    testProject.setDefinition(
+        new CpsFlowDefinition(
+            loadResource(getClass(), "noNamespaceDeclarativePipeline.groovy"), true));
+    copyTestFileToDir(
+        getClass(),
+        jenkinsRule.jenkins.getWorkspaceFor(testProject).getRemote(),
+        TEST_DEPLOYMENT_MANIFEST);
+
+    WorkflowRun run = testProject.scheduleBuild2(0).waitForStart();
+    assertNotNull(run);
+    jenkinsRule.assertBuildStatus(Result.SUCCESS, jenkinsRule.waitForCompletion(run));
+    dumpLog(LOGGER, run);
+
+    kubectlDelete(
+        jenkinsRule.createLocalLauncher(),
+        jenkinsRule.jenkins.getWorkspaceFor(testProject),
+        TEST_DEPLOYMENT_MANIFEST,
+        "deployment",
+        "nginx-deployment",
+        null);
```

Comment:
Prefer use of Optional instead of null.

Reply:
This reflects the value that would happen in the case where where namespace is omitted. However, the behavior is identical with using the empty string, so that may work here to avoid null. Would that work?